lib/gcloud/jsondoc/markup_helper.rb

```diff
@@ -30,12 +30,20 @@ def linkify(name, title)
         code_obj = YARD::Registry.resolve object.namespace, name, true
         if code_obj.nil?
           name
+        elsif code_obj.type == :method
+          path = code_obj.namespace.path.gsub("::", "/").downcase
+          method_type = get_method_type code_obj
+          method_id = "#{code_obj.name}-#{method_type}"
```

Comment:
This is a purely cosmetic thing, but I don't love appending `-instance` and `-class` to the method ids. Could we change this so class methods were prepended by `.`, and instance methods were left unchanged? This should work since `.` isn't a legal character for a method, so it would still be unique.

```ruby
method_id = code_obj.name
method_id = ".#{method_id}" if method_type == "class"
```

That would give us URLs like `#/docs/master/gcloud?method=bigquery` and `#/docs/master/gcloud?method=.bigquery` instead of `#/docs/master/gcloud?method=bigquery-instance` and `#/docs/master/gcloud?method=bigquery-class`